test_data/asm/full_pil_constant.asm

```diff
@@ -1,5 +1,11 @@
-degree 1;
```

Comment:
I think this was created by leo. His point was to create a minimal example. Is a pc now a hard requirement?

Reply:
It was me lol. Yeah some things were breaking if I didn't have a pc. I can also try to fix them instead of changing the tests like this.